spidermon/contrib/actions/slack/__init__.py

```diff
@@ -85,27 +86,26 @@ def _get_user_id(self, username):
         return user["id"] if user else None
 
     def _get_users_info(self):
-        return dict(
-            [
-                (member["name"].lower(), member)
-                for member in self._api_call("users.list")["members"]
-            ]
-        )
-
-    def _api_call(self, method, **kwargs):
-        response = self._client.api_call(method, **kwargs)
-
-        has_errors = not response.get("ok")
-        if has_errors:
-            error_msg = response.get("error", {}).get("msg", "Slack API error")
-            logger.error(error_msg)
-
-        if isinstance(response, six.string_types):  # slackclient < v1.0
-            response = json.loads(response)
-        return response
+        try:
+            return dict(
+                [
+                    (member["name"].lower(), member)
+                    for member in self._client.users_list()["members"]
+                ]
+            )
+        except SlackApiError as e:
+            assert e.response["ok"] is False
+            assert e.response["error"]
+            logger.error(e.response["error"])
 
     def _get_user_channel(self, user_id):
-        return self._api_call("im.open", user=user_id)["channel"]["id"]
+        try:
+            response = self._client.conversations_open(users=[user_id])
+            return response["channel"]["id"]
+        except SlackApiError as e:
+            assert e.response["ok"] is False
+            assert e.response["error"]
+            logger.error(e.response["error"])
 
     def _send_user_message(
```

Comment:
[user_channel](https://github.com/scrapinghub/spidermon/pull/265/files#diff-56dc9c5d7e8074bc42c77ec35c9c09c2L117) needs to be the same as `user_id`, or else the message will not be sent to the user.